examples/port-2-alibaba2-connection/README.md

```diff
@@ -0,0 +1,69 @@
+# Fabric Port to Fabric Alibaba Profile Connection
+
+This is a step-by-step guide for creating a connection using the Equinix Terraform Provider and the Alibaba Terraform Provider.
+By following these steps, you will be able to:
+
+1. Setup a connection through Equinix Fabric.
+2. Configure and accept the connection in the Alibaba Portal.
+3. Manage resources effectively using Terraform.
+4. Perform cleanup operations efficiently
+
+### Step by Step Instructions for Fabric Port to Fabric Alibaba Profile Connection
+
+#### 1. Create Connection from Equinix Terraform Provider
+
+This example shows how to leverage the [Fabric Port Connection Module](https://registry.terraform.io/modules/equinix/fabric/equinix/latest/submodules/port-connection)
+to create a Fabric Connection from a Fabric Port to Fabric Alibaba Service Profile.
+
+It leverages the Equinix Terraform Provider, the Alibaba Terraform Provider, Fabric Port Connection
+Module and various Alibaba resources to setup the connection based on the parameters you have provided to this example; or based on the pattern
+you see used in this example it will allow you to create a more specific use case for your own needs. 
+
+See the example usage (provided after the steps) for details on how to use this example.
+
+**Note:** The `connection_name` must follow either one of these patterns:
+
+* A unique `connection_name` with atmost 24 characters.  
+`connection_name = var.connection_name`
+
+* A `connection_name` with atmost 12 characters combined with a random string of 12 characters:  
+  `connection_name = "${var.connection_name}${random_string.random.result}"`
+
+**Result of this step:**
+* A unique `connection_name` is generated by appending a 12-character random string to the provided name.
+* A Fabric Port to Alibaba Profile Connection is successfully created and displayed in the Fabric Portal.
+* An Express Connect Physical Connection resource is created and becomes visible in the Alibaba Portal.
+
+#### 2. Formulate main.tf and output.tf Files
+Use the following resources and data sources from the example (place them as comments initially):
+
+`data "alicloud_express_connect_physical_connections" {}`
+
+` resource "alicloud_express_connect_virtual_border_router" {}`
+
+`resource "null_resource" {}`
+
+`output alicloud_express_connect_virtual_border_router {}`
+
+`output alicloud_express_connect_virtual_border_router_id {}`
+
+#### 3. Accept the Connection Request
+Manually accept the connection request in the Alibaba Portal for the created physical connection.
+
+#### 4. Create the Virtual Border Router (VBR) Resource
+Remove the commented code to create the VBR resource
+
+**Result of this step:**
+The VBR resource is created successfully and linked to the physical connection.
+
+#### 5. [Cleanup] Delete Resources
+1. Delete the VBR using terraform
```

Comment:
This isn't explaining exactly what will happen and is causing the user to lookup information to understand it. Tell them to:
1. Run a `terraform destroy` which will delete the Alibaba VBR and give an error on the Equinix Fabric Connection which matches the one you noted.
2. Go to Alibaba Portal to manually delete the connection; (You will need to explain if this automatically deletes connection on Equinix side or if there's an additional 3rd step).